packages/frontend-2/components/server-management/DeleteUserDialog.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <LayoutDialog v-model:open="isOpen" max-width="sm" :title="title" :buttons="buttons">
+    <div class="flex flex-col gap-6">
+      <p>
+        Are you sure you want to
+        <strong>permanently delete</strong>
+        the selected user?
+      </p>
+      <div v-if="user" class="flex items-center gap-2">
+        <UserAvatar :user="user" />
+        {{ user.name }}
+      </div>
+      <p>
+        This action
+        <strong>cannot</strong>
+        be undone.
+      </p>
+    </div>
+  </LayoutDialog>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import { LayoutDialog } from '@speckle/ui-components'
+import { UserItem } from '~~/lib/server-management/helpers/types'
+
+const emit = defineEmits<{
+  (e: 'update:open', val: boolean): void
+}>()
+
+const props = defineProps<{
+  title: string
+  open: boolean
+  user: UserItem | null
+  buttons?: Array<{ text: string; props: Record<string, unknown>; onClick: () => void }>
```

Comment:
Why should these be passed in from the parent component? It's always going to be the same set of buttons